templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -46,6 +46,7 @@ contents:
     }
 
     persist_nm_conn_files() {
+      update_nm_conn_files br-ex phys0
```

Comment:
conflict: remove mentions of br-ex 1 and phys1 as 4.8 has no support for secondary bridge in configure-ovs.